app/views/admin/users/actions/_banish.html.erb

```diff
@@ -0,0 +1,20 @@
+<div id="banish-for-spam">
+  <div class="flex flex-col gap-4">
+    <% if @user.comments.where("created_at < ?", 100.days.ago).empty? && @user.created_at < 100.days.ago %>
+      <p><strong>This is extremely destructive.</strong> Banishing will delete all the user's existing content and change their username to <em>@spam_###</em>.</p>
+      <p><strong>Do not do this lightly.</strong></p>
+      <%= form_for(@user, url: banish_admin_user_path(@user), html: { method: :post, onsubmit: "return confirm('Are you sure? This is extremely destructive and irreversible. Banishing will delete all articles and turn their username into @spam_###')" }) do %>
+        <button class="c-btn c-btn--primary c-btn--destructive">Banish User for spam</button>
+      <% end %>
+    <% elsif current_user.super_admin? || current_user.support_admin? %>
+      <p><strong>This is not a new user.</strong> You are only allowed to take this action because you are a <strong>super admin or a support admin.</strong></p>
```

Comment:
**_Quick suggestion (same as below!):_**
```suggestion
      <p><strong>This is not a new user.</strong> You are only allowed to take this action because you are a <strong>Super Admin or a Support Admin.</strong></p>
```